packages/utils/telemetry-utils/src/utils.ts

```diff
@@ -30,3 +31,38 @@ export function logIfFalse(
 	logger.send(newEvent);
 	return false;
 }
+
+/**
+ * sampletext
+ */
+export function createSampledLoggerSend(
+	logger: ITelemetryBaseLogger,
+	shouldSampleEventCallback: () => boolean,
+) {
+	const mc = createChildMonitoringContext({ logger });
```

Comment:
I think using `loggerToMonitoringContext()` here might more accurately represent what we're trying to accomplish (just reading config from the object that was passed in). This calls it internally but also creates a `ChildLogger` out of `logger`, which we don't need in this case since we're going to directy call `.send()` on the logger we received.

Reply:
Sounds good. Updated.